packages/guideline-blocks-settings/src/hooks/useAttachments.tsx

```diff
@@ -42,5 +45,51 @@ export const useAttachments = (appBridge: AppBridgeBlock, assetId: string) => {
         onAttachmentReplace,
         onAttachmentsSorted,
         attachments,
+        appBridge,
```

Comment:
Where is this used?

Reply:
It's being used so that the attachment button in the toolbar has access to the correct appBridge component without needing to add an additional prop.